src/MoBi.Presentation/MenusAndBars/ContextMenus/ContextMenuForSimulation.cs

```diff
@@ -41,11 +46,16 @@ public IContextMenu InitializeWith(ObjectBaseDTO dto, IPresenter presenter)
          {
             createEditItem(simulation),
             createRenameItem(simulation),
+            createConfigure(simulation).AsGroupStarter()
+         };
 
-            createConfigure(simulation).AsGroupStarter(),
-            createUpdate(simulation),
-            createCommit(simulation),
+         if (simulation.BuildingBlocks().Any(hasChanges))
+            _allMenuItems.Add(createUpdate(simulation));
+         if (simulation.OriginalQuantityValues.Any())
```

Comment:
run resharper in this file

Reply:
It did not make any changes for these if statements FYI. I just added a line.

Reply:
Sure. Still run resharoer because the file is now off lol.